### PR TITLE
TextView: Adds missing method documentation

### PR DESCRIPTION
**Pull request by jleandroperez**
### Details:
Added missing methods documentation, and minor changes.

Needs Review: @diegoreymendez 
Thanks in advance!

### CODE REVIEW

File: Aztec/Classes/TextKit/TextView.swift [hunk 1]
```diff
@@ -611,11 +613,13 @@ open class TextView: UITextView {
         formatter.toggleAttribute(inTextView: self, atRange: range)
     }
 
-    /// Refresh Lists attributes when text is deleted in the specified range
+
+    /// Toggles the Blockquote Style, whenever the deleted text is at the Location 0, which is 
+    /// also known as the "Beginning of the text"
```
Comment (diegoreymendez) [inline Aztec/Classes/TextKit/TextView.swift]: I think it'd be better if we bring back the original description, and concatenate your new documentation below it as a more detailed description of what happens.

The reason being that the original description described what the external usage for this class is, and when external actors would want to use this method, whereas the new documentation explains internal details about what it really does.

Comment (diegoreymendez) [inline Aztec/Classes/TextKit/TextView.swift]: Also consider changing to "...whenever the deleted text is at location 0."  The "also known as the beginning of the text" part should be pretty clear anyway.

**Comment by jleandroperez** [comment]
@diegoreymendez ready for another look.

Thanks!!!

**Comment by jleandroperez** [comment]
Thanks Diego!

**Pull request merged**